notebooks/QA_extraction_generation.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
<p>Does this context actually contain answer to this prompt?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Yes it does. This is the tricky part. In the above case, I knew the placement of the context. I am trying to figure out if there is any way to detect the context from the given document based on a given query, and later feed that context into the text generation model .</p>

Reply:
Cool Cool!